components/colorarea/index.css

```diff
@@ -74,8 +74,11 @@ governing permissions and limitations under the License.
 
 .spectrum-ColorArea-handle {
   transform: translate(calc(var(--mod-colorarea-width, var(--spectrum-colorarea-width)) - var(--spectrum-colorarea-border-width)), 0);
-  top: 0;
-  left: 0;
+  inset-block-start: 0;
+
+  [dir="rtl"] & {
+    inset-inline-end: 0;
+  }
```

Comment:
saw that these physical properties snuck though, updated them while looking at the ColorArea with in the ColorWheel